apps/system/js/action_menu.js

```diff
@@ -120,23 +120,18 @@
   };
 
   ActionMenu.prototype._appendDefault = function() {
-    var defaultChoice = document.createElement('label');
-    defaultChoice.setAttribute('class', 'pack-checkbox');
+    var defaultChoice = document.createElement('gaia-checkbox');
+    defaultChoice.className = 'inline';
+    defaultChoice.configureClass();
```

Comment:
Really? This API design is questionable. But it's not related to this patch though.